sbol3/document.py

```diff
@@ -279,6 +279,22 @@ def _guess_format(self, fpath: str):
             rdf_format = 'nt11'
         return rdf_format
 
+    # Return standard extensions when provided the document's file format
+    def file_extension(self, file_format: str) -> str:
```

Comment:
Let's make this a `staticmethod` since it doesn't use `self`, and add a docstring:
```suggestion
    @staticmethod
    def file_extension(file_format: str) -> str:
        """Return standard extensions when provided the document's file format

        :param file_format: The format of the file
        :return: A file extension, including the leading '.'
        """
```